### ISSUE TITLE
VITASDK not set in macOS Catalina Error

### ISSUE THREAD

**Issue by Pikachuxxxx** [vitasdk/vdpm#53]
So I tried following the steps on MacOS Catalina 10.15.1(19B88)(Dev-beta). I did add the lines 
`export VITASDK=/usr/local/vitasdk
export PATH=$VITASDK/bin:$PATH`
![bashrc setup](https://user-images.githubusercontent.com/45527623/73371670-01e14400-42dc-11ea-8645-9b37db6ff8bf.png)

 at the end of .bashrc as prompted by the terminal 
![pls add vitasdk temp prompt](https://user-images.githubusercontent.com/45527623/73371765-2e955b80-42dc-11ea-9a6b-c2c02a240ff8.png)
 and tried running the the following commands as shown in the screenshots and yet nothing seems to work. It still says I haven't set the path.

![terminal commands](https://user-images.githubusercontent.com/45527623/73371671-01e14400-42dc-11ea-84fa-9214c2906b23.png)

**Comment by frytkisasmaczne** [comment]
either tell zsh to load run exports and stuff from .bashrc or put the exports in .zshrc instead of .bashrc since you're using zsh shell not bash

**Comment by d3m3vilurr** [comment]
@frytkisasmaczne could be right. it's just shell problem. but we should give the better suggestion in running script.

**Comment by Pikachuxxxx** [comment]
Thanks for pointing it out and helping me resolve it. As Catalina using the new Zsh as the default shell I think proper instructions to use the .zshrc along with .bashrc update would clear any issues or confusions for new explorers.

**Comment by default** [comment]
Hi, everyone 👋. I really am not familiar with how to properly work with Terminal and scripts so forgive me if I appear stupid at times. 

I'm on macOS Catalina 10.15.3 and, from what I understood, `.zshrc` is, basically, a file with instructions, run whenever I launch Terminal (a config of sorts). I created such file (since it was not present before) in my "home" (~) directory and put the suggested `export` lines in there. Just to make sure, I also made it `echo` so I get a confirmation that instructions from that file were executed. 

And, on launch, I get that confirmation, yes, and I also put all two of the `export` lines inside `.bash_profile` and `.bashrc`. Those are the only 2 lines in all 3 files (not counting `echo`'s). However, no matter how many times I tried (restarting Terminal each time), running the `bootstrap-vitasdk.sh` script always fails with a requirement to put `export`'s into the `.bashrc`. And, therefore, `install-all.sh` also fails, stating that "$VITASDK is not set".

What am I doing wrong and how can I bypass this?
Thank you in advance :)

**Comment by default** [comment]
@d3m3vilurr could you, please, help me resolve this issue 🙏

**Comment by frytkisasmaczne** [comment]
does it say anything when you put `echo $VITASDK` into terminal?

**Comment by Pikachuxxxx** [comment]
Can you share some screenshots that'd help us understand the problem better and do check properly if you're in the zsh or bash shell?

**Comment by default** [comment]
@Pikachuxxxx @frytkisasmaczne Here's the whole process with `echo $VITASDK` at the end:
<img width="571" alt="Failed installation attempt" src="https://user-images.githubusercontent.com/11653648/78011498-52146b00-734c-11ea-886d-9d4c2fc5bddf.png">

My `.zshrc`, just in case. You can see its `echo` on the second row in Terminal above.
<img width="384" alt="Screenshot 2020-03-31 at 12 37 57" src="https://user-images.githubusercontent.com/11653648/78011674-8f78f880-734c-11ea-8c93-51f355157ca4.png">

Not sure if it is at all relevant, but I've upgraded to macOS 10.15.4 now but everything's, pretty much, the same

P.S. @frytkisasmaczne French fires are, indeed, rather tasty ;)

**Comment by frytkisasmaczne** [comment]
I don't think this is an error, it's just a reminder to set the exports. To be sure that it installed correctly run `vitasdk-update` or clone `https://github.com/vitasdk/samples` and run 
`cmake . && make` in one of the sample project directories, let's say in `cd hello_world`.

**Comment by Pikachuxxxx** [comment]
I think a 'process successful' message would be great.

**Comment by default** [comment]
@frytkisasmaczne 
`vitasdk-update` is also failing, after a download:
<img width="571" alt="vitasdk-update attempt" src="https://user-images.githubusercontent.com/11653648/78257045-03103680-7502-11ea-92ac-0a880c6b37fd.png">
And what about the installation script: running `sudo ./install-all.sh` results in a "$VITASDK is not set"? :c

**Comment by default** [comment]
@frytkisasmaczne Ok, 2 things:

1. `cmake . && make` on "hello_world" worked as intended (I assume)
<img width="697" alt="Compilation and building" src="https://user-images.githubusercontent.com/11653648/78258845-79ae3380-7504-11ea-8933-440ec1a0d3d6.png">

2. Neither `./install-all.sh` nor `vitasdk-update` succeeded:
<img width="697" alt="Screenshot 2020-04-02 at 17 05 51" src="https://user-images.githubusercontent.com/11653648/78258955-ae21ef80-7504-11ea-8e16-d5891012f328.png">

So something's definitely wrong

**Comment by loszer** [comment]
1.change the /usr/local/vitasdk dir's right to 775
2.change the dir's owner to current user
3.mkdir a "arm-vita-eabi" under  /usr/local/vitasdk
Next, "./install-all.sh" work for me

**Comment by Engylizium** [comment]
1. chmod 775 /usr/local/vitasdk
2. chown "USERNAME" /usr/local/vitasdk
3. cd /usr/local/vitasdk 
    mkdir arm-vita-eabi
4. ./install-all.sh
NAH $VITASDK is not set

**Comment by keiichiro1993** [comment]
@Engylizium 
I had the exact same issue with all the workarounds so edited install-all.sh as follows and it worked fine
![image](https://user-images.githubusercontent.com/4514810/200159406-012ddeb5-7b0a-49d2-a63a-6eceee2a8e1c.png)

**Comment by frangarcj** [comment]
Closing — the current installer doesn't push you toward `.bashrc` specifically anymore, it just prints the two `export` lines at the end and leaves where to put them up to your shell. Doesn't fully solve "how do I persist an env var," but the specific confusion this thread is about (bash vs zsh, Catalina) doesn't apply to what the script does today.
